payment_processors/braintree_processor/models/payment_processors.py

```diff
@@ -6,12 +6,11 @@
 from django.utils import timezone
 from django_fsm import TransitionNotAllowed
 
+from .payment_methods import BraintreePaymentMethod
+from ..views import BraintreeTransactionView
```

Comment:
Well I don't think it's that bad. In Django projects, the views are usually on the same level with the models, but since in this case models is a package, I just had to go a level up.